solutions/swb-reference/integration-tests/tests/multiStep/awsAccount.test.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { AccountService, CreateAccountData } from '@aws/workbench-core-accounts';
+import { Account } from '@aws/workbench-core-accounts/lib/models/accounts/account';
+import { resourceTypeToKey } from '@aws/workbench-core-base';
+import DynamoDBService from '@aws/workbench-core-base/lib/aws/helpers/dynamoDB/dynamoDBService';
+import ClientSession from '../../support/clientSession';
+import { AccountHelper } from '../../support/complex/accountHelper';
+import Setup from '../../support/setup';
+import RandomTextGenerator from '../../support/utils/randomTextGenerator';
+import Settings from '../../support/utils/settings';
+
+describe('multiStep awsAccount integration test', () => {
+  const setup: Setup = new Setup();
+  const settings: Settings = setup.getSettings();
+  let adminSession: ClientSession;
+
+  beforeEach(async () => {
+    // expect.hasAssertions();
```

Comment:
Why this was commented out? Seems like a legit case, if test passes it will have at least one assertion

Reply:
Missed this during cleanup 🙏